packages/office-ui-fabric-react/src/components/SwatchColorPicker/SwatchColorPicker.classNames.ts

```diff
@@ -0,0 +1,76 @@
+import { memoizeFunction } from '../../Utilities';
+import {
+  ITheme,
+  IPalette,
+  mergeStyleSets,
+  getFocusStyle
+} from '../../Styling';
+
+export interface ISwatchColorClassName {
+  themeClass?: string | undefined;
+}
+
+export const getClassNames = memoizeFunction((
+  theme: ITheme,
+  color: string
+): ISwatchColorClassName => {
+  const themedColor: string | undefined = getSwatchThemeColor(theme, color);
+
+  return mergeStyleSets({
+    themeClass: [themedColor ?
+      {
+        fill: themedColor
+      } :
+      undefined
+    ]
+  });
+
+});
+
+function getSwatchThemeColor(theme: ITheme, color: string) {
+  const palette: IPalette = theme.palette;
+  switch (color) {
```

Comment:
How is this function any different than (color: string) => color? Not sure I follow what's going on here

Reply:
Sure it returns undefined if there isn't a color in the theme for it, but this will quickly get out of date.